src/keycodeMap.ts

```diff
@@ -1,8 +1,17 @@
-export default {
-  'control+p': 'togglePause',
-  'control+t': 'putTimestamp',
-  'control+s': 'saveNote',
-  'control+shift+right': 'videoForward',
-  'control+shift+left': 'videoBackward',
-  'control+shift+i': 'captureFrame',
+import fromPairs from './utils/fromPairs';
+
+const keycodeOfCommandName: { [s: string]: string } = {
+	toggle_pause: 'Ctrl-p',
+	mark_selection_as_timestamp: 'Ctrl-t',
+	turn_text_to_timestamp: 'Ctrl-y',
+	put_timestamp: 'Ctrl-Shift-t',
+	seek_to_timestamp: 'Ctrl-g',
+	capture_frame: 'Ctrl-i',
+	debug_print: 'Ctrl-d',
 };
+
+const commandNameOfKeycode = fromPairs(
+	Object.entries(keycodeOfCommandName).map(([commandName, keycode]) => [keycode, commandName])
```

Comment:
`Object.entries(keycodeOfCommandName).map(([commandName, keycode]) => [keycode, commandName])`
or
`Object.entries(keycodeOfCommandName).map(([cmd, key]) => [key, cmd])` 
?

Try to make code easier to read.

Reply:
Making this change?